session/src/test/java/org/apache/iotdb/session/IoTDBSessionIT.java

```diff
@@ -794,7 +811,7 @@ private void insertInObject() throws IoTDBConnectionException, StatementExecutio
     }
   }
 
-  private void insert() throws IoTDBConnectionException, StatementExecutionException {
+  private void insertInObjectList() throws IoTDBConnectionException, StatementExecutionException {
     String deviceId = "root.sg1.d1";
     List<String> measurements = new ArrayList<>();
```

Comment:
move the insertInObject() method to its caller. There are too many methods in this file..